radio/src/lua/api_lcd.cpp

```diff
@@ -377,7 +379,12 @@ static int luaLcdDrawRectangle(lua_State *L)
   int w = luaL_checkinteger(L, 3);
   int h = luaL_checkinteger(L, 4);
   unsigned int flags = luaL_optunsigned(L, 5, 0);
+  #if defined(PCBHORUS)
+  int t = luaL_optinteger(L, 6, 0);
```

Comment:
The `thickness` parameter is unsigned, so I would prefer `unsigned int t =  tluaL_optunsigned()`